src/porepy/numerics/fv/mpsa.py

```diff
@@ -1329,13 +1253,10 @@ def reconstruct_displacement(g, subcell_topology, eta=None, eta_at_bnd=False):
         g: Grid
         subcell_topology: Wrapper class for numbering of subcell faces, cells
             etc.
-        eta (float, range=[0,1]): Optional. Parameter determining the point at which the
-            displacement is evaluated. If eta is not given the method will call
-            fvutils.determine_eta(g) to set it.
-        eta_at_bnd (bool): Optional, defaults to False.
-            In MPSA eta is only enforced at internal faces, while is always set to
-            eta=0 at the boundary. eta_at_bnd=True will override this behaviour,
-            and give the displacement at the points given by eta, also for the boundary.
+        eta (float or ndarray, range=[0,1]): Optional. Parameter determining the point
+            at which the displacement is evaluated. If eta is a nd-array it should be on
+            the size of subcell_topology.num_subfno. If eta is not given the method will
```

Comment:
This also means that etas that vary on the sub-face level must be set according to the numbering in that array? Sound like one really has to know what one is doing, but it has to be that way, I suppose. The alternative of leaving it to fvutils is fully acceptable.

Reply:
Yes, eta has to be set according to SubcellTopology. I can't really think of any other way of setting this. I guess if you want to set the continuity points on the subcells it can be required that you know what you are doing